### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by XenGi**
I updated the code to adhere to the Python coding standards a bit better and also switched to more modern approaching for example string manipulation. I also removed some comments that basically said what the code underneath said :)
Nice little module btw! :+1:

### CODE REVIEW

File: goss.py [hunk 1]
```diff
@@ -94,55 +101,44 @@ def main():
...
-    test_file_path = module.params['path']  # test file path
+    test_file_path = module.params['path']
     output_format = module.params['format']  # goss output format
     output_file_path = module.params['output_file']
     goss_path = module.params['goss_path']
     vars_path = module.params['vars_path']
 
-    if test_file_path is None:
-        module.fail_json(msg="test file path is null")
-
```
Comment (ggrossetie) [inline goss.py:105]: Why did you remove this test?

File: goss.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
-#!/usr/bin/env python
```
Comment (ggrossetie) [inline goss.py:1]: I guess the code is now only compatible with Python >= 3? It's not really an issue but we should state it explicitly since Ansible is still working with Python 2 right?

**Comment by ggrossetie** [comment]
>  updated the code to adhere to the Python coding standards a bit better

I think we should add a linter to enforce a coding standard. We could potentially enable GitHub Actions to run the linter on pull requests. I don't have a strong opinion so feel free to suggest a linter library.

File: goss.py (hunk 1 above)
Comment (XenGi) [reply goss.py:105]: Because the module already makes the path mandatory.

File: goss.py (hunk 2 above)
Comment (XenGi) [reply goss.py:1]: Python 2 hit end of life 01.01.2020. no one should use anymore.

**Comment by XenGi** [comment]
I'm pretty happy with flake8 for linting and checking my code. It has lots of plugins.

**Comment by ggrossetie** [comment]
Thanks @XenGi (and sorry for the delay)
I've enabled GitHub Actions with flake8: https://github.com/indusbox/goss-ansible/blob/master/.github/workflows/ci.yml

Next step would be to add unit tests and why not a few integration tests 😉
